packages/esm-service-queues-app/src/index.ts

```diff
@@ -156,6 +156,29 @@ export const patientSearchWorkspace = getAsyncLifecycle(() => import('./patient-
   moduleName,
 });
 
+export const totalPatientsInQueueMetrics = getAsyncLifecycle(
+  () => import('./queue-table-metrics-extensions/total-patients-in-queue.extension'),
+  {
+    featureName: 'total-patients-in-queue-metrics',
+    moduleName,
+  },
+);
+
+export const waitingStatusQueueMetrics = getAsyncLifecycle(
+  () => import('./queue-table-metrics-extensions/waiting-status.extension'),
+  { featureName: 'waiting-status-queue-metrics', moduleName },
+);
+
+export const inServiceStatusQueueMetrics = getAsyncLifecycle(
+  () => import('./queue-table-metrics-extensions/in-service-status.extension'),
+  { featureName: 'in-service-status-queue-metrics', moduleName },
+);
+
+export const finishedServiceStatusQueueMetrics = getAsyncLifecycle(
+  () => import('./queue-table-metrics-extensions/finished-service-status.extension'),
+  { featureName: 'finished-service-status-queue-metrics', moduleName },
+);
+
```

Comment:
I agree that there should be a single component configurable in the way @mseaton describes, using extension config.

However, I'd also suggest that these three extensions should also still be here, as thin wrappers around the configurable component which provide the correct default configuration for the RefApp. So the RefApp ships with these three attached, and then if someone wants to swap them out they just use the generic queue metric extension.

It would be nice if we had a better way of dealing with this kind of thing (attaching an extension to a slot, possibly multiple times, with configuration set for the attachment), but this is how we've been doing it so far.